test/integration/targets/docker_host_facts/tasks/test_host_facts.yml

```diff
@@ -0,0 +1,249 @@
+---
+- block:
+  - name: Get info on Docker host
+    docker_host_facts:
+    register: output
+
+  - name: assert reading docker host facts when docker is running
+    assert:
+      that:
+         - 'output.docker_host_facts.Name is string'
+         - 'output.docker_containers_list is not defined'
+         - 'output.docker_networks_list is not defined'
+         - 'output.docker_volumes_list is not defined'
+         - 'output.docker_images_list is not defined'
+         - 'output.docker_disk_usage is not defined'
+
+  - name: Create random container name prefix
+    set_fact:
+      cname_prefix: "{{ 'ansible-test-%0x' % ((2**32) | random) }}"
+      cnames: []
+      dnetworks: []
```

Comment:
Why do you create these lists (and populate one with content) if you don't use it for cleanup?